docs/csharp/language-reference/configure-language-version.md

```diff
@@ -0,0 +1,95 @@
+> set the value the same in all build configurations, but you need to
+> set it explicitly for each build configuration, or select "All Configurations"
+> when you modify this setting. You'll see the following in your csproj file:
+>
+>```xml
+> <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+>  <LangVersion>latest</LangVersion>
+></PropertyGroup>
+>
+> <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+>  <LangVersion>latest</LangVersion>
+> </PropertyGroup>
+> ```
+>
+
+## Edit the csproj file
+
+You can set the language version in your **.csproj** file. Add an element like the following:
+
+```xml
+<PropertyGroup>
+   <LangVersion>latest</LangVersion>
+</PropertyGroup>
+```
+
+The value `latest` uses the latest minor version of the C# language. Valid values are:
+
+|Value|Meaning|
+|------------|-------------|
+|default|The compiler accepts all valid language syntax from the latest major version that it can support.|
+|ISO-1|The compiler accepts only syntax that is included in ISO/IEC 23270:2003 C# (1.0/1.2) |
+|ISO-2|The compiler accepts only syntax that is included in ISO/IEC 23270:2006 C# (2.0) |
+|3|The compiler accepts only syntax that is included in C# 3.0 or lower.|
+|4|The compiler accepts only syntax that is included in C# 4.0 or lower.|
+|5|The compiler accepts only syntax that is included in C# 5.0 or lower.|
+|6|The compiler accepts only syntax that is included in C# 6.0 or lower.|
+|7|The compiler accepts only syntax that is included in C# 7.0 or lower.|
+|7.1|The compiler accepts only syntax that is included in C# 7.1 or lower.|
+|7.2|The compiler accepts only syntax that is included in C# 7.2 or lower.|
+|7.3|The compiler accepts only syntax that is included in C# 7.3 or lower.|
+|latest|The compiler accepts all valid language syntax that it can support.|
+
+The special strings `default` and `latest` resolve to the latest major (C# 7.0)
+and minor (C# 7.3) language versions installed on the build machine, respectively.
+
+## Configure multiple projects
+
+You can create a **Directory.build.props** file that contains the `<LangVersion>` element to configure multiple directories. You typically do that in your solution directory. Add the following to a **Directory.build.props** file in your solution directory:
+
+```xml
+<Project>
+ <PropertyGroup>
+   <LangVersion>7.3</LangVersion>
+ </PropertyGroup>
+</Project>
+```
+
+Now, builds in every subdirectory of the directory containing that file will use C# version 7.3 syntax. For more information, see the article on [Customize your build](/visualstudio/msbuild/customize-your-build.md).
+
+## Set the langversion compiler option
+
+You can use the `-langversion` command line option. For more information, see the article on the [-langversion](../language-reference/compiler-options/langversion-compiler-option.md) compiler option. You can see a list of the valid values by typing  `vbc -langversion:?`.
```

Comment:
command line option -> command-line option

vbc? shouldn't be csc here? should you tell that they need to type on the developer command prompt for vs?

Reply:
and, updated command-line in the VB specific doc.